internal/mocks/authelia_ctx.go

```diff
@@ -7,12 +7,12 @@ import (
 	"testing"
 	"time"
 
-	"github.com/golang/mock/gomock"
 	"github.com/sirupsen/logrus"
 	"github.com/sirupsen/logrus/hooks/test"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/valyala/fasthttp"
+	"go.uber.org/mock/gomock"
```

Comment:
Are we not using MockGen for this file?

Reply:
We are, the original was replaced with the uber one.

Reply:
Yeah, I just noticed that all other MockGen files have commentary at the start of the file yet this one does not.